frontend/packages/integration-tests-cypress/views/list-page.ts

```diff
@@ -34,6 +48,13 @@ export const listPage = {
     countShouldBe: (count: number) => {
       cy.get(`[data-test-rows="resource-row"`).should('have.length', count);
     },
+    clickFirstLinkInFirstRow: () => {
+      cy.get(`[data-test-rows="resource-row"]`)
+        .first()
+        .find('a')
+        .first()
+        .click({ force: true }); // after applying row filter, resource rows detached from DOM according to cypress, need to force the click
```

Comment:
I'd like to understand why this is better. Are we sure we're clicking on the right link after the table has been filtered?

Reply:
Definitely clicking on the correct link.  I removed the `{force: true}` and get: 
![image](https://user-images.githubusercontent.com/12733153/108863974-c2108d00-75bf-11eb-901d-09c9c7dfad5e.png)

Here is the [Learn More...](https://docs.cypress.io/guides/references/error-messages.html#cy-failed-because-the-element-you-are-chaining-off-of-has-become-detached-or-removed-from-the-dom) link from the above screenshot.

I think our Filter asyncronously filters and re-renders the table without reloading screen.  Cypress has documentation on it, but I haven't yet found a solution/workaround.

Reply:
> Definitely clicking on the correct link.

I mean, if it's removed from the DOM, is it really the correct link? :)

Is the page updating from underneath the tests while `.first().find('a').first()` is being evaluated? Is there a way to avoid the use of `first` here?